ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetBucketView.java

```diff
@@ -62,7 +63,7 @@ public String render(final GetBucketResult br) {
                 new ASCIITableHeader("ETag", ASCIITable.ALIGN_RIGHT)};
     }
 
-    private String nullGuard(String message) {
+    private String nullGuard(final String message) {
```

Comment:
Same comment as above about consolidating this into a single table view util function